src/components/common/PrivateRoute.js

```diff
@@ -2,21 +2,17 @@ import React from 'react';
 import { Route } from 'react-router-dom';
 import { withAuthenticationRequired } from '@auth0/auth0-react';
 import Sidebar from './Sidebar/Sidebar';
+import { Spin } from 'antd';
```

Comment:
Nice addition here, ensuring the user has some feedback to let them know the app is working on serving a page up to them.

Another alternative that can be quite smooth is [Ant Design's Skeleton component](https://ant.design/components/skeleton/), something for your consideration any other time you have some component or changing state that needs some time to do its job!